homework/generate-sequence/generateSequence.hpp

```diff
@@ -2,6 +2,11 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    
+    std::vector<int> vec;
+
+    for(auto element=1; element <= count; element++ ){
+        vec.push_back(step*element);
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..